src/modules/authentication/components/Login/Login.tsx

```diff
@@ -1,5 +1,123 @@
-function Login() {
-	return <div></div>;
+import { useContext, useState } from 'react'
+import { Link, useNavigate } from 'react-router-dom'
+import './login.css'
+import logo from '../../../../assets/images/logo.png'
+import { useForm } from 'react-hook-form'
+import { emailValidation, PasswordValidation } from '../../../../validations'
+import { AUTH_URLS, axiosInstance, HEADERS } from '../../../../api'
+import { toast } from 'react-toastify'
+import { AuthContext } from '../../../../context/AuthContext'
+import { LoginFormData } from '../../../../interface/AuthInfo/AuthInfo'
+
+const Login = () => {
+	let {saveLoginData} = useContext(AuthContext)
+
+	
+	let{
+		register,
+		handleSubmit,
+		formState:{errors , isSubmitting}
+	} = useForm<LoginFormData>()
+	let navigate = useNavigate()
+
+	let onSubmitData =async (data : LoginFormData) => {
+		try {
+			let response = await axiosInstance.post(AUTH_URLS.login ,data , HEADERS)
+			console.log(response)
+	   		navigate('/dashboard')
+			toast.success('Login Successfully')
+			localStorage.setItem('token',response?.data?.token)	
+			saveLoginData()
+		} catch (error) {
+			console.log(error)
```

Comment:
We need to handle the api call error, to help users know what is going wrong

Reply:
done